syntax="proto3";

import "google/protobuf/any.proto";

package data;

message MsgReq {
    // 方便以后扩展
}

message MsgRsp {
    int32           code  = 1;
    bytes           info  = 2;
}

message Body {

    oneof msg_type {
        MsgReq  msg_req = 1;
        MsgRsp  msg_rsp = 2;
    }

    google.protobuf.Any service_msg = 3;    // 扩展各个服务消息
}

